schemas/context/person-name.schema.json

```diff
@@ -36,7 +36,7 @@
           "description":
             "Normally an abbreviation of a persons *title*, *honorific*, or *salutation*.\nThe `courtesyTitle` is used in front of full or last name in opening texts.\ne.g Mr. Miss. or Dr J. Smith.\n"
         },
-        "xdm:name": {
+        "xdm:fullName": {
```

Comment:
We use `xdm:name` for the proper name in other places, I don't see reason to deviate here.

Reply:
@trieloff So what would you put in "name" a persons preferred name or their legal(called when in trouble or by ones mother) name.

Reply:
I feel like "fullName" makes it clearer what should go in here.

Reply:
@cdegroot-adobe I'd put in:

> The full name of the person, in writing order most commonly accepted in the language of the name.

Exactly as the description says. I think the `title="Full Name"` makes things clear enough and makes things more consistent as we don't have `*name` properties elsewhere. `fullName` is a special case that in my mind does not have enough justification. (Using the full name in the `name` property is something I absolutely support)